src/components/Navbar/Navbar.tsx

```diff
@@ -16,27 +15,16 @@ export type NavbarProps = BoxProps & {
   resumeLink?: string;
 };
 
-export const NavbarComponent = ({
-  withoutBlogPage,
-  type,
-  fullName,
-  image,
-  position,
-  resumeLink,
-  ...props
-}: NavbarProps & { withoutBlogPage: boolean }) => {
+export const NavbarComponent = ({ type, fullName, image, position, resumeLink, ...props }: NavbarProps) => {
   const mobileProps = { fullName, image, position };
   const tabletProps = { fullName, image, position, resumeLink };
 
-  if (type === 'mobile') return <NavbarMobile {...mobileProps} {...props} withoutBlogPage={withoutBlogPage} />;
-  if (type === 'tablet') return <NavbarTablet {...tabletProps} {...props} withoutBlogPage={withoutBlogPage} />;
+  if (type === 'mobile') return <NavbarMobile {...mobileProps} {...props} />;
+  if (type === 'tablet') return <NavbarTablet {...tabletProps} {...props} />;
 
-  return <NavbarDesktop {...props} withoutBlogPage={withoutBlogPage} />;
+  return <NavbarDesktop {...props} />;
 };
 
 export const Navbar = (props: NavbarProps) => {
```

Comment:
I think, now there is no need to separate `Navbar` and `NavbarComponent`. I would suggest to move all rendering logic to the `Navbar` and remove the `NavbarComponent`

Reply:
Done 😄